math_keyboard/lib/src/widgets/math_field.dart

```diff
@@ -338,6 +340,15 @@ class _MathFieldState extends State<MathField> with TickerProviderStateMixin {
     Overlay.of(context).insert(_overlayEntry!);
   }
 
+  /// Removes the keyboard overlay from the screen.
+  ///
+  /// Keep in mind: it does not remove the focus from the field.
+  void _closeKeyboard() {
+    _overlayEntry?.remove();
+    _overlayEntry = null;
+    _keyboardSlideController.reverse();
```

Comment:
We should only remove the `overlayEntry` when the animation is finished. Otherwise, it will not play